docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -278,10 +279,10 @@ the ESPW.
 ## Deployment into a Kubernetes cluster
 
 These commands administer a deployment of the central components ---
-the kcp server, PKI, and the central KubeStellar components --- in a
-Kubernetes cluster that will be referred to as "the hosting cluster".
-These commands are framed as "kubectl plugins" and thus need to be
-explicitly or implicitly given a kubeconfig file for the hosting
+the space provider server, PKI, and the central KubeStellar components 
```

Comment:
"server" is too simplistic for KubeFlex, is has more stuff than just a "server". s/space provider server/space provider/

This text also needs to talk about the space abstraction layer, right?